analytics/pubstack/helpers/json.go

```diff
@@ -0,0 +1,95 @@
+package helpers
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/prebid/prebid-server/analytics"
+)
+
+// JsonifyAuctionObject helpers to serialize auction into json line
+func JsonifyAuctionObject(ao *analytics.AuctionObject, scope string) ([]byte, error) {
+	type alias analytics.AuctionObject
+	b, err := json.Marshal(&struct {
```

Comment:
Is there a reason you define the alias type, rather than simply declaring that the value of alias in the struct is of the appropriate type?

Have you tested that this will marshal correctly with `alias` unexported?